src/egon/data/datasets/vg250/__init__.py

```diff
@@ -278,14 +280,14 @@ def add_metadata():
 def nuts_mview():
 
     db.execute_sql_script(
-        os.path.join(os.path.dirname(__file__), "vg250_lan_nuts_id_mview.sql")
```

Comment:
This must not be changed to the working directory as the scripts is part of the repository